<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewInitSet</class>
 <widget class="QDialog" name="NewInitSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>启动画面设置</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>启动Logo画面号：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_logoNum">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>311</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>其他</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>21</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>初始画面号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>53</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前配方组：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>19</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_peifang">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>83</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前配方号：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_peifang_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>111</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>权限不足显示锁定图标</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>311</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>待机设置</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QComboBox" name="comboBox_standbyModel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>待机方式：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>待机画面号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>待机亮度：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_standbyLight">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>待机保护时间：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_standbyTime">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>283</x>
      <y>113</y>
      <width>20</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>分</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_standbyLogin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>待机注销登录</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_standbyNum">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_sure">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>73</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>73</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpbtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>400</y>
     <width>73</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
